msvc/mono.props

```diff
@@ -17,7 +17,8 @@
     <MONO_USE_STATIC_LIBMONO>false</MONO_USE_STATIC_LIBMONO>
   </PropertyGroup>
   <PropertyGroup Label="MonoDirectories">
-    <top_srcdir>$(MSBuildProjectDirectory)/..</top_srcdir>
+    <MonoSourceLocation Condition="'$(MonoSourceLocation)' == '' ">..</MonoSourceLocation>
+    <top_srcdir>$(MSBuildProjectDirectory)/$(MonoSourceLocation)</top_srcdir>
     <MONO_DIR>$(top_srcdir)</MONO_DIR>
```

Comment:
can't we just simplify top_srcdir, MonoSourceLocation and MONO_DIR to:

`<MONO_DIR Condition="'$(MONO_DIR)' == '' ">$(MSBuildProjectDirectory)/..</MONO_DIR>`

or is top_srcdir needed for something else?